middleware/middleware-1.3.24.md

```diff
@@ -0,0 +1,64 @@
+We've received reports about an issue that lead to failing TAR exports on some Linux distributions due to a missing dependency. The issue has been resolved.
+
+## Bug fix: Large numbers were incorrectly formatted in TSE process data
+We've resolved an issue in the Middleware that could lead to incorrectly formatted TSE process data in some rare cases when the overall amount of a single receipt exceeded € 1000. Due to this error, a thousand separator was inserted, although this is not provided for in the DSFinV-K. 
+
+## Bug fix: SCU switch was not working in BYODC installations
+Due to an issue in our _Bring your own DataCenter_ solution, SCU switch operations could not be executed in some scenarios, leading to errors during the process. We've resolved this issue and further increased the systems stability to prevent similar errors in the future.
+
+## Bug fix: FileUploadHelper failed to upload data on legacy OSs
+We've resolved an issue where the FileUploadHelper used an outdated TLS version on legacy Operating Systems like Windows Server 2008, which lead to failing data uploads.
+
+## Stability improvement: Added clear error messages for deleted fiskaly v2 test TSEs
+Due to fiskaly's practice of deleting version 2 test TSEs weekly (each Sunday), many of our users who were not aware of this practice got confused by the fact that their Middleware test instances stopped working after each weekend. We've improved the log messages and errors the Middleware displays in this case to reduce the investigation effort to detect this issue for new users.
+
+## Stability improvement: Improved Android lifecycle management
+We've received note that some Android devices tend to kill the Middleware Launcher when running into low memory or battery scenarios. While this is a valid scenario for Android apps, our Middleware was unable to recover from this situation without a manual full restart of the service. 
+
+We've improved the overall lifecycle handling of the underlying Android service, so that the App is now able to automatically recover when the service is killed by the Android OS. Additionally, we've added improved logging and simplified our service management to enhance the Developer experience when implementing and testing the Middleware on Android.
+
+## Documentation: Improved process documentation for DSFinV-K exports
+We've received and incorporated some feedback about the DSFinV-K process documentation templates added last month, including a clear changelog, details about timestamps, and several smaller terminology clarifications.
+
+## Affected packages
+Packages not listed here were not updated, as we decided to not increase the version of unchanged packages. All packages with versions greater or equal to 1.3.1 are compatible with each other (it is e.g. possible to use _fiskaltrust.Middleware.SCU.Swissbit.1.3.1_ with the new queue packages).
+
+- _fiskaltrust.Middleware.Queue.EF v1.3.24_
+- _fiskaltrust.Middleware.Queue.MySQL v1.3.24_
```

Comment:
Do you want to mention the postgres prototype?

Reply:
Rather not yet, as it's not publicly usable now 👍